plugins/circleci/src/circleci-config.ts

```diff
@@ -115,6 +117,19 @@ export default abstract class CircleCiConfigHook extends Hook<CircleCIState> {
 
   async commitInstall(state: CircleCIState): Promise<void> {
     const rawConfig = await this.getCircleConfig()
+
+    if (rawConfig && isNotToolKitConfig(rawConfig)) {
+      throw new ToolKitError(
+        `Your project has an existing CircleCI config file which doesn't contain a ${styles.heading(
+          'tool-kit'
+        )} workflow.
+If you would like a Tool Kit configured CircleCI config file to be generated for you please delete your existing CircleCI ${styles.filepath(
+          'config.yml'
+        )} and re-run:
+${styles.code('npx dotcom-tool-kit --install')}`
```

Comment:
**nitpick**: imo the newline here makes the suggestion harder to follow, especially as code snippets are already greyed out. seeing as it's just a single terminal invocation I think it's okay to be written inline

```suggestion
        )} and re-run ${styles.code('npx dotcom-tool-kit --install')}`
```